src/styles/Flipper.module.scss

```diff
@@ -0,0 +1,37 @@
+.flipperWrapper {
+    display: flex;
+    position: relative;
+    align-items: center;
+    justify-content: center;
+    width: 100vw;
+}
+
+.imageWrapper {
+  position: relative;
+  text-align: center;
+}
+
+.text {
+  position: absolute;
+  top: 5%;
+  padding: 0 1vw;
+  min-width: 100%;
+  font-size: 1.2vw;
+  font-weight: 500;
+}
+
+.textIsClosed {
+    font-size: 1.8vw;
```

Comment:
Can you increase the font-size to totally fill the available space on different smaller screen widths with a media query please?  Do your best to make it as visible as possible. 
![image](https://user-images.githubusercontent.com/6663002/227783218-d5ff5945-b12e-4c2e-8540-00672f9ab751.png)